components/next_steps_view/next_steps_tips.tsx

```diff
@@ -150,6 +154,39 @@ export default function NextStepsTips(props: Props) {
         nonMobileTips = (
             <>
                 <Card expanded={true}>
+                    <div className='Card__image'>
+                        <img
+                            src={tipsNextStepsImage3}
+                        />
+                    </div>
```

Comment:
@catalintomai Can we place the `Card_image` divs inside the `Card_body` for each card? That way it can inherit the padding for the body content and aligns perfectly.

Reply:
@abhijit-singh , not really. Here's what I'm getting: 
<img width="1357" alt="Screen Shot 2021-02-21 at 11 19 22 PM" src="https://user-images.githubusercontent.com/56169943/108675560-d2c7e280-749b-11eb-8ba1-c2e06b4fe7ea.png">